### ISSUE TITLE
bundler crashes when video is embedded in mdx

### ISSUE THREAD

**Issue by arindamlahiri** [ksoichiro/rehype-img-size#2]
### Error
![image](https://user-images.githubusercontent.com/54589973/139236602-34d28f69-50cd-48fe-bde9-ef80e8ac734c.png)

### MDX
`![MDXJS' website flicker before toggling dark mode on refresh](/blog/the-perfect-dark-mode/perfect-dark-mode-mdxjs-flicker.mp4)`

### generated HTML element
![image](https://user-images.githubusercontent.com/54589973/139220022-192ce275-2aa4-4c84-93a8-29d5e237c0a7.png)

The plugin is not able to calculate dimensions for a video file, hence the bundler errors out.

## Suggested solution
At [Line 15:](https://github.com/ksoichiro/rehype-img-size/blob/c42ec33d3927b7f4e02b1fec6bc13c959e2d687a/index.js#L15)

```js
if (node.tagName === 'img' && !node.properties.src.endsWith('mp4')) {
```

We could even maintain a list of video formats and compare it against that list instead of just mp4.

**Comment by arindamlahiri** [comment]
I could make a PR for this. Let me know

**Comment by ksoichiro** [comment]
Sorry for my late response, and thank you for your PR! It's merged 🚀

**Comment by ksoichiro** [comment]
Released the change as v0.0.2. Thank you for your report!
